Tournament/tournament.py

```diff
@@ -341,15 +351,37 @@ async def addMatch( self, a_plyrs: List[str] ) -> None:
             matchCategory = discord.utils.get( self.guild.categories, name="Matches" ) 
             if len(matchCategory.channels) >= 50:
                 matchCategory = category=discord.utils.get( self.guild.categories, name="More Matches" ) 
-            newMatch.VC    = await matchCategory.create_voice_channel( name=f'{self.tournName} Match {newMatch.matchNumber}', overwrites=overwrites ) 
+                
+            game_name: str = str(self.tournName) + "Match" + str(newMatch.matchNumber)
```

Comment:
this discards the joining spaces, it used to be like this:
```suggestion
            game_name: str = f'{self.tournName} Match {newMatch.matchNumber}'
```

Reply:
nice catch :)